internal/fs/fs_test.go

```diff
@@ -124,7 +124,7 @@ func TestHasFilepathPrefix_Files(t *testing.T) {
 		want   bool
 	}{
 		{existingFile, filepath.Join(dir2), false},
-		{nonExistingFile, filepath.Join(dir2), true},
+		{nonExistingFile, filepath.Join(dir2), false},
```

Comment:
Now, since this becomes `false`, let's add another testcase where `HasFilepathPrefix` returns `true`.